javaslang/src/test/java/javaslang/collection/StringTest.java

```diff
@@ -216,12 +216,18 @@ public void replace() {
 
     @Test
     public void split() {
-        assertThat(CharSeq.of("123").split("2")).isEqualTo(Array.of(CharSeq.of("1"), CharSeq.of("3")));
+        //noinspection deprecation
+        assertThat(CharSeq.of("123").split("2")).isEqualTo(new CharSeq[] { CharSeq.of("1"), CharSeq.of("3") }); // TODO deprecated
```

Comment:
tests for old methods, with disabled deprecation warning (the build won't pass otherwise)